src/xdocs/checks.xml

```diff
@@ -966,6 +966,10 @@
               VariableDeclarationUsageDistance</a></td>
             <td>Checks the distance between declaration of variable and its first usage.</td>
           </tr>
+          <tr>
```

Comment:
I may be wrong, but it seems that the checks in this file should be sorted alphabetically (by check name). Please move it to right after `MagicNumber`.

Reply:
done